docs/registry/training-scripts.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Browse ML Model Training Scripts"
```

Comment:
```suggestion
title: "Training Scripts for ML Models in the Registry"
```
The registry page should probably just link to https://docs-test.viam.dev/3590/services/ml/training-scripts/ and cover versioning information?

Reply:
In that case it sounds like I should just not add this sub-page at all right?

Reply:
Add it and cover versioning